cli/exchange/org.go

```diff
@@ -0,0 +1,165 @@
+package exchange
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/open-horizon/anax/cli/cliutils"
+	"github.com/open-horizon/anax/i18n"
+	"net/http"
+)
+
+type ExchangeOrgs struct {
```

Comment:
Can you use the GetOrganizationResponse struct from the exchange package? or is there some reason you need the map entries to be an interface{} type?

Reply:
The GetOrganizationResponse does not contain all of the fields in the org such as HeartbeatIntervals, type, and tags. I want to show all this info to the user for the list command.

Reply:
For the list function, GetOrganizationResponse doesn't give all the information because it some attributes has "omitEmpty". list should show all the fields from the org. By the way, `hzn exchange user` has a similar struct so I based it off of that.

Reply:
ok